src/server.js

```diff
@@ -389,12 +412,25 @@ function initializeClient(socket, client, token, lastMessage) {
 			}
 		});
 
-		socket.emit("sign-out");
+		_.map(client.attachedClients, (attachedClient, socketId) => {
+			if (attachedClient.token !== tokenToSignOut) {
+				return;
+			}
+
+			const socketToRemove = manager.sockets.of("/").connected[socketId];
+
+			socketToRemove.emit("sign-out");
+			socketToRemove.disconnect();
+		});
+
+		sendSessionList(client);
```

Comment:
Why do you need to send the list when logging out?

Reply:
So that client sees the updated session list.